pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -357,7 +391,7 @@ func getManagedKeys(secret *v1.Secret, fieldOwner string) ([]string, error) {
 		if dataFields == nil {
 			continue
 		}
-		df, ok := dataFields.(map[string]string)
+		df, ok := dataFields.(map[string]interface{})
```

Comment:
this was a bug in my setup. changing to interface{} to provide better coverage (as we only look for the keys anyways)